docker_checker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM rocker/r-devel
+MAINTAINER karl.forner@quartzbio.com
```

Comment:
Do you want to remain the maintainer for the docker stuff or should I be the maintainer?